boxoffice/models/item.py

```diff
@@ -64,6 +64,11 @@ def get_by_category(cls, category):
     def quantity_available(self):
         return self.quantity_total - self.get_confirmed_line_items.count()
 
+    @property
+    def is_available(self):
+        """Checks if an item has a current price and has a positive quantity_available"""
+        return bool(self.current_price() and self.quantity_available > 0)
```

Comment:
`current_price()` can't be `0`. In case of a free item, `current_price()` will return a `Price` object, whose `amount` will be `0`.